test/k8s-integration/main.go

```diff
@@ -75,12 +76,21 @@ func init() {
 func main() {
 	flag.Parse()
 
-	if !*inProw {
+	if !*inProw && !*useGKEManagedDriver {
 		ensureVariable(stagingImage, true, "staging-image is a required flag, please specify the name of image to stage to")
 	}
 
+	if *useGKEManagedDriver {
+		ensureVariableVal(deploymentStrat, "gke", "deployment strategy must be GKE for using managed driver")
+		ensureFlag(doDriverBuild, false, "driver build flag will be ignored when using GKE managed driver")
```

Comment:
more accurate to say that the flag must be false, as that's what you're checking.

Reply:
Ack